cmake/modules/SetupCompiler.cmake

```diff
@@ -89,11 +95,11 @@ macro(dyad_add_c_flags MY_FLAGS)
 endmacro()
 
 dyad_add_cxx_flags(CMAKE_CXX_FLAGS
-  -fPIC -Wall -Wextra -pedantic -Wno-unused-parameter -Wnon-virtual-dtor
+  -Wall -Wextra -pedantic -Wno-unused-parameter -Wnon-virtual-dtor
```

Comment:
@JaeseungYeom

Reply:
@JaeseungYeom @hariharan-devarajan I slightly misspoke yesterday.

I did not replace `-fPIC` with something else. I just removed it here (and below for C) because we were already using the CMake compiler-independent version of specifying it. So, the manual use of `-fPIC` here is redundant.

This is the line where the CMake-specific variable is set: https://github.com/TauferLab/dyad/blob/00fe8812a60ffeab37ce5814b3d6947d45d37031/CMakeLists.txt#L195